README.md

```diff
@@ -11,39 +11,39 @@ Hier findest du die technischen Präsentationen des Workshops:
 * Sonntag: [Einführung in Django](https://docs.google.com/a/djangogirls.org/presentation/d/1R92A025MCB-srM2xnczdeC_nWozMlqh3tlksSlJP494/edit?usp=sharing)
 
-Während dem Workshop werden wir die folgenden Themen bearbeiten.
+Während des Workshops werden wir die folgenden Themen bearbeiten:
 
 ### 1. Die Kommandozeile \(Samstag Morgen\)
 
-Die nächsten zwei Tage werden wir ganz viel mit der Kommandozeile arbeiten. Gehe durch das Kapitel ["Einführung in die Kommandozeile"](https://tutorial.djangogirls.org/de/intro_to_command_line/) um dich damit vertraut zu machen und die Basics zu lernen.
+Die nächsten zwei Tage werden wir häufig mit der Kommandozeile arbeiten. Gehe durch das Kapitel ["Einführung in die Kommandozeile"](https://tutorial.djangogirls.org/de/intro_to_command_line/), um dich damit vertraut zu machen und die Basics zu lernen.
 
 ### 2. Deine erste statische Webseite \(Samstag Morgen\)
 
 Jetzt wirst du deine [erste statische Webseite erstellen und auf GitHub veröffentlichen](./static-website/README.md).
 
 ### 3. Einführung in Python \(Samstag Nachmittag\)
 
-Jetzt wird es richtig nerdig \(aber keine Sorge, dein Coach wird dir helfen\), wir lassen unsere statische Website ruhen und folgen dem Kapitel [Introduction to python](https://tutorial.djangogirls.org/en/python_introduction/) aus dem Django Girls Tutorial. Wir werden Python morgen brauchen um die nächste Herausforderung zu bewältigen ne dynamische Webseite zu erstellen.
+Es wird richtig nerdig \(aber keine Sorge, dein Coach wird dir helfen\)! Wir lassen unsere statische Webseite ruhen und folgen dem Kapitel [Introduction to python](https://tutorial.djangogirls.org/en/python_introduction/) aus dem Django Girls Tutorial. Wir werden Python morgen brauchen, um die nächste Herausforderung zu bewältigen: eine dynamische Webseite erstellen.
 
 ### 4. Django: Dein Blog ist eine dynamische Webseite \(Sonntag\)
 
-Jetzt tauchen wir richtig ins Django Girls Tutorial ein. Ab dem Kapitel [What is Django?](https://tutorial.djangogirls.org/en/django/) geht 's los. Dein Django Blog ist ein Projekt unabhängig von deiner statische Webseite. Es muss also in einem anderen Ordner gestartet werden. Auch in Github wird es an einem anderen Ort abgelegt als das was du bis jetzt gemacht hast. Github kan nur statische Seiten selber darstellen, darum werden wir unsere dynamische Webseite noch an einem anderen Ort im Internet publizieren. Wir werden aber immer noch Github benutzen um unsere Code zu speichern, diese Mal aber nur als Zwischenschritt. Von hier werden wir dann einen anderem Service \(Pythonanywhere\) benutzen um die Seite darzustellen. Das klingt kompliziert, und das ist es auch. Deine dynamische Seite online zu bringen eine grosse Leistung. Aber keine Sorge, es ist alles detailliert im Django Girls Tutorial beschrieben und wird werden dir dabei helfen!
```

Comment:
"GitHub" ohne bestimmten Artikel verwenden. (Also "in GitHub" anstatt "im GitHub".)

Tendenziell würde ich sogar eher "auf GitHub" als "in GitHub" schreiben, da es sich um eine Internetplattform handelt. (So wie "in Twitter" und "in Facebook" nicht falsch sind, aber "auf Twitter" und "auf Facebook" eigentlich üblicher.)

Reply:
gemacht: 9d68cc81843aeddc370bf6d77520bd553e35ad5c